packages/extensions/src/app/(dashboard)/layout.tsx

```diff
@@ -0,0 +1,60 @@
+"use client";
+import * as React from "react";
+import { Box } from "@mui/material";
+import {
+  OIDCSecure,
+  Dashboard,
+} from "@dirac-grid/diracx-web-components/components";
+import {
+  ApplicationsProvider,
+  DiracXWebProviders,
+} from "@dirac-grid/diracx-web-components/contexts";
+import { usePathname, useRouter, useSearchParams } from "next/navigation";
+import { applicationList } from "@/gubbins/applicationList";
+import { defaultSections } from "@/gubbins/defaultSections";
+
+// Layout for the dashboard: setup the providers and the dashboard for the applications
+export default function DashboardLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  // Get the pathname, router and search params from the next/navigation package, needed for navigation and routing
+  const pathname = usePathname();
+  const router = useRouter();
+  const searchParams = useSearchParams();
+
+  // A custom logo URL can be used for the dashboard
+  const customLogoURL =
+    "https://mattermost.web.cern.ch/files/oktn8gxjobrb9gwkznx3hx3z7w/public?h=VpJiHpv03q76Pv6KqX90y-dkGxOResdO9xFOa4JsMr4";
```

Comment:
Is there any reason not to include it in `public`?

Reply:
Not any particular reason, I'll add it.